jobs/JJOB_POST

```diff
@@ -23,7 +26,7 @@ case $FCSTMODEL in
        ;;
    "MOM6CICE5")
        # TODO: Keep something from fcst?
-       if [ -d $RUNCDATE/fcst ]; then rm -rf $RUNCDATE/fcst; fi
+       #if [ -d $RUNCDATE/fcst ]; then rm -rf $RUNCDATE/fcst; fi
```

Comment:
Could you explain this change?

Reply:
@flampouris I can undo this change. When I run fcst_only case, this line delete fcst run directory and no figures and so on...

Reply:
Yes, this is a bug. If you have a fix ready, put it in now, otherwise we will tackle to a future pr

Reply:
@flampouris Let's revisit later about this issue. You can merge.

Reply:
@flampouris @jkbk2004  For when we revisit this later: I think if we move the netcdf output files from MOM6 & CICE to a "COMOUT" or "save" place, and then have the post point to them there that this will be resolved with that.  Because it will allow you to delete the fcst directory then (although this should maybe be "fcst/mem##" so you aren't deleting another directory where a fcst is still running in the ensemble.

Reply:
@JessicaMeixner-NOAA this is good point! I will revisit this issue in next PR!